android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/register/components/PatientListItem.kt

```diff
@@ -100,11 +100,7 @@ fun PatientRow(
               }
 
               Text(
-                text = (it.valuePrefix ?: "") + it.value,
-                color =
-                  Color(
-                    android.graphics.Color.parseColor(it.properties?.value?.color ?: "#000000")
-                  ),
+                text = " Last Test - ${it.lastDateAdded}",
```

Comment:
not sure if "Last Test - ##/##/##" would need to be handle for MLS, we can add in String.xml if so